apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -131,7 +131,7 @@ const VideoPreviews: React.FC<{
   );
 };
 
-const getNativeElementsList = (elements: SBEnumType['value']): JSX.Element => {
+const getNativeElementsList = (elements: SBEnumType['value']): React.ReactElement => {
```

Comment:
using `ReactElement` directly - this is not public facing story which shows source

Reply:
https://github.com/microsoft/fluentui/pull/34938